packages/apps/human-app/server/src/modules/oracle-discovery/model/oracle-discovery.model.ts

```diff
@@ -107,14 +115,27 @@ export class DiscoveredOracle {
   executionsToSkip = 0;
 
   constructor(props: DiscoveredOracleCreateProps) {
-    Object.assign(this, props);
-    this.registrationNeeded = props.registrationNeeded || false;
-    this.stakedAmount = this.stakedAmount.toString();
-    this.lockedAmount = this.lockedAmount.toString();
-    this.withdrawnAmount = this.withdrawnAmount.toString();
-    this.slashedAmount = this.slashedAmount.toString();
-    this.amountJobsProcessed = this.amountJobsProcessed.toString();
-    this.lockedUntilTimestamp = this.lockedUntilTimestamp.toString();
+    this.id = props.id;
+    this.address = props.address;
+    this.chainId = props.chainId;
+    this.registrationNeeded = props.registrationNeeded ?? undefined;
+    this.role = props.role ?? undefined;
+    this.url = props.url ?? undefined;
+    this.name = props.name ?? undefined;
```

Comment:
Actually, these 3 are required for `DiscoveredOracle` instance and we ensure it when doing checks in `checkExpectationsOfDiscoveredOracle`, so IMU we can keep them defined